physics/module_bl_mynn.F90

```diff
@@ -7066,14 +7087,14 @@ SUBROUTINE DDMF_JPL(kts,kte,dt,zw,dz,p,              &
 !      call Poisson(1,NDOWN,kts+1,kte,ENTf,ENTi)
 
 
-      ! entrainent: Ent=Ent0/dz*P(dz/L0)
-      do i=1,NDOWN
-         do k=kts+1,kte
-!            ENT(k,i)=real(ENTi(k,i))*Ent0/(ZW(k+1)-ZW(k))
-            ENT(k,i) = 0.002
-            ENT(k,i) = min(ENT(k,i),0.9/(ZW(k+1)-ZW(k)))
-         enddo
-      enddo
+!      ! entrainent: Ent=Ent0/dz*P(dz/L0)
```

Comment:
Consider removing the commented out code and relying on git history to recover these statements if it is every necessary to.

Reply:
I probably will remove it at some point. That subroutine is under construction. It will be gone by the next PR.